### ISSUE TITLE
Making package UAP compatible

### ISSUE THREAD

**Issue by ameno-** [takenet/textc-csharp#1]
What would it take to make this package work with a UAP app? I've tried adding it in portable library, but the error I get is `Takenet.Textc 0.2.7 is not compatible with DNXCore,Version=v5.0.`

**Comment by andrebires** [comment]
The project doesn't support the .NET Core currently, but it is definitely something that we will consider in a near future. But you can always send us a pull request :)

**Comment by ameno-** [comment]
I'm still fairly new to the Microsoft stack so I don't really know where to start, but if you can point me in the right direction, I'd be happy to do this.

What are some good google searches that'll help me find more info about adding support for .NET Core to a nuget package?

[I've found this article](https://oren.codes/2015/07/29/targeting-net-core/) about targeting .NET Core, is this what TextC needs?

**Comment by andrebires** [comment]
Yes, you will need a recompiled version of the source, targeting the .NET Core version.

You should take a look at https://dotnet.github.io/getting-started/ and https://blogs.msdn.microsoft.com/dotnet/2016/02/10/porting-to-net-core/. The project now supports only .NET 4.5.1 but I believe that will be very straightforward the port to the .NET Core.
